cogs/search_utils/regex.py

```diff
@@ -0,0 +1,150 @@
+from re import compile, I, X
+
+nono_re = compile(
+    r"""tranny|
+faggot|
```

Comment:
Use correct string indentation.
**This is just an example, you need to re-indent every word here, don't just commit this**
```suggestion
    r"""tranny|
    faggot|
```